src/redux/actions/mapActions.js

```diff
@@ -1,26 +1,17 @@
 // import {UPDATE_LAT,UPDATE_LONG,ZOOM} from "./mapType";
 import {UPDATE_CORDINATES} from "../types/mapType";
+import {UPDATE_MAPSTYLE} from "../types/mapType";
 
-/*export const updateLat = () => {
-    return {
-        type:UPDATE_LAT
-    }
-}
-export const updateLONG = () => {
-    return {
-        type:UPDATE_LONG
-}
-}
-export const zoom = () => {
-    return {
-        type:ZOOM
-    }
-}*/
 export const UpdateCordinates=(lng,lat,zoom) => {
     return {
         type:UPDATE_CORDINATES,
         payload:{lng,lat,zoom}
     }
 }
+export const UpdateMapStyle=()=>{
```

Comment:
Generally the naming convention we are following is:
- camelCase (first letter in lowerCase) for all methods & files of type actions, reducers and pretty much all pure JS files.
- Capitalized for react component files & classes